docs/src/content/docs/getting-started.mdx

```diff
@@ -126,4 +126,4 @@ See the growing list of guides in the sidebar for help adding content and custom
 
 If your answer cannot be found in these docs, please visit the [full Astro Docs](https://docs.astro.build) for complete Astro documentation. Your question may be answered by understanding how Astro works in general, underneath this Starlight theme.
 
-You can also get help in the [Astro Discord](https://astro.build/chat/) from our active, friendly community! You can post questions in our `#support` forum or visit our dedicated `#starlight` channel to discuss current development and more!
+You can also check for any known [Starlight issues on GitHub](https://github.com/withastro/starlight/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc), and get help in the [Astro Discord](https://astro.build/chat/) from our active, friendly community! Post questions in our `#support` forum with the "starlight" tag, or visit our dedicated `#starlight` channel to discuss current development and more!
```

Comment:
is the query intended or just from the github extension?

Reply:
It's from the Github default link, and I didn't hate it enough to change it. :smile: But, not opposed to just the straight up link!

Reply:
@TheOtterlord , propose the change for credit!